src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, Drawable {
```

Comment:
We should not be able to create new Figure because we don't khow how it looks like. Figure is something abstract
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```